doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -1,87 +1,54 @@
-* Reusable sub-process, which is independent from its parent process.
-* Event sub-process, which is only triggered on a start event or a timer.
+The following subprocess types are supported in {PRODUCT}:
 
-Note that any sub-process type can be a multi-instance sub-process.
+* Embedded subprocess, which is a part of the parent process execution and shares its data
+* Ad-Hoc subprocess, which has no strict element execution order
+* Reusable subprocess, which is independent from its parent process
+* Event subprocess, which is only triggered on a start event or a timer
+* Multi-instance subprocess
 
-.Embedded Sub-Process
+In the following example, the Place Order subprocess checks whether there is sufficient stock to place the order and updates the stock information if the order can be placed. The customer is then notified through the main process based on whether or not the order was placed.
 
+image:BPMN2/subprocess.png[]
 
-An embedded sub-process encapsulates a part of the process.
+.Embedded subprocess
 
-It must contain a start event and at least one end event. Note that the element allows you to define local sub-process variables, that are accessible to all elements inside this container.
+An embedded subprocess encapsulates a part of the process. It must contain a start event and at least one end event. Note that the element enables you to define local subprocess variables that are accessible to all elements inside this container.
 
 [[_adhoc_sub_process]]
-.AdHoc Sub-Process
-COMMENT: edit 
-"`An Ad-Hoc Sub-Process is a specialized type of Sub-Process that is a group of Activities that have no REQUIRED sequence relationships. A set of Activities can be defined for the Process, but the sequence and number of performances for the Activities is determined by the performers of the Activities. footnote:[Business Process Model and Notation (BPMN). Version 2.0, OMG Document Number: formal/2011-01-03 http://www.omg.org/spec/BPMN/2.0]`"
+.AdHoc subprocess
 
+An ad-hoc subprocess or process contains a number of embedded inner activities and is intended to be executed with a more flexible ordering compared to the typical routing of processes. Unlike regular processes, an ad-hoc subprocess does not contain a complete, structured BPMN2 diagram description, for example from start event to end event. Instead the ad-hoc subprocess contains only activities, sequence flows, gateways, and intermediate events. An ad-hoc subprocess can also contain data objects and data associations. The activities within the ad-hoc subprocesses are not required to have incoming and outgoing sequence flows. However, it is possible to specify sequence flows between some of the contained activities. When used, sequence flows provide the same ordering constraints as in a regular process. To have any meaning, intermediate events must have outgoing sequence flows and they can be triggered multiple times while the ad-hoc subprocess is active.
```

Comment:
"description, for example, from" or "description, such as from". But it seems odd either way. Not really an example of what it's talking about, to me.

"Instead, the ad hoc" (comma, and no hyphen). Search "ad-hoc" and replace with "ad hoc". Technically not hyphenated even when used as a modifier, according to MW.

"However, you can specify sequence ..." Along with avoiding "there is/are", same with the empty "it is".